src/server/test-suites/transformers/test-suite-to-summary.js

```diff
@@ -18,7 +18,7 @@ function transformTestSuiteToSummary(testSuite) {
     }).trim()
   )
 
-  return {
+  const list = {
```

Comment:
I'd expect Sonar is complaining about this being a redundant local var?

Reply:
think this was me debugging.